soapui/src/main/java/com/eviware/soapui/impl/wsdl/support/wsdl/WsdlLoader.java

```diff
@@ -129,11 +131,13 @@ private InvalidDefinitionException makeInvalidDefinitionException(String url, Ex
 
     private String readCleanWsdlFrom(String url) throws Exception {
         String content = XmlUtils.createXmlObject(load(url)).xmlText();
-
         if (SoapUI.getSettings().getBoolean(WsdlSettings.TRIM_WSDL)) {
             content = content.trim();
         }
-        return PropertyExpansionRemover.removeExpansions(content);
+        if (!UISupport.handleDefinitionPropertyExpansions(url, content)) {

```

Comment:
Is this really safe for runners?

Reply:
Yes, the headless mode is handled inside ``handleDefinitionPropertyExpansions``.